Unreal/CarlaUnreal/Plugins/Carla/Source/Carla/Autoware/Game/AutowareGameModeBase.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2024 Computer Vision Center (CVC) at the Universitat Autonoma de Barcelona (UAB). This work is licensed under the terms of the MIT license. For a copy, see <https://opensource.org/licenses/MIT>.
```

Comment:
```suggestion
// Copyright (c) 2024 Computer Vision Center (CVC) at the Universitat Autonoma
// de Barcelona (UAB).
//
// This work is licensed under the terms of the MIT license.
// For a copy, see <https://opensource.org/licenses/MIT>.
```
Consistent with other licenses